packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -362,10 +388,12 @@ Then('I see tab {string}', (search: string) => {
   cy.findAllByRole('tab').first().should('be.visible').contains(search);
 });
 
-Then('I see {string}', (message: string) => {
-  cy.findByRole('document')
-    .contains(new RegExp(escapeRegExp(message), 'i'))
-    .should('exist');
+Then('I see {string}', doesDocumentContainText);
+
+Then('I see {string} files with random names', (count: string) => {
+  for (let i = 1; i <= parseInt(count); i++) {
```

Comment:
Does it make sense to consider cases where `count` could resolve to NaN, since it's a string? I guess if tests have an error they're (hopefully) not getting merged. What do you think?

Reply:
If it results in NaN say we input "two" instead of "2", the file simply won't get uploaded.
So in the following step we won't see `All files successfully uploaded` status and the test would fail.

We could do something like this but i was thinking it might be a bit overkill
```
  if (isNaN(numberOfFiles)) {
    throw new Error(`Invalid count: "${count}" is not a valid number`);
  }
```

Reply:
You love to kill birds right? You can not kill a 🥔 potato , just kill it with two stones🤣🍕

Reply:
More emotion 🐟🐟🐟

Reply:
You want jon snow them Ashwin 😉

Reply:
May be a bottle of soju will fix this♥️🐟😉